search/search_quality/merge_samples.py

```diff
@@ -0,0 +1,186 @@
+import argparse
+import copy
+import json
+
+
+def read_samples(path):
+    with open(path, "r") as f:
+        lines = f.readlines()
+        return [json.loads(x) for x in lines]
+
+
+def doubles_equal(a, b):
+    EPS = 1e-6
+    return abs(a - b) < EPS
+
+
+def points_equal(a, b):
+    for coord in ["x", "y"]:
+        if not doubles_equal(a[coord], b[coord]):
+            return False
+    return True
```

Comment:
return doubles_equal(a['x'], b['x']) and doubles_equal(a['y'], b['y']) ? It seems less confusing imo

Reply:
done